transports/azure-service-bus-netstandard/compatibility.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Compatibility
+summary: Describes the requirements for backward compatibility with ASB transport
+component: ASBS
+tags:
+ - Azure
+reviewed: 2018-06-19
+---
+
+## Backward compatibility
+
+The Azure Service Bus .Net Standard transport is backward compatible with the Azure Service Bus transport under a certain set of preconditions.
+
+### Preconditions
+
+#### Forwarding topology
+
+The Azure Service Bus .Net Standard transport only supports the forwarding topology, an entity layout where a shared set of topics is used for publishing between the endpoints.
+
+#### Single namespace
+
+The Azure Service Bus .Net Standard transport only supports a single namespace at the moment.
```

Comment:
Please remove "at the moment". We don't want to write anythign we can't commit to. I'd rather say less with what we've got, than promise more of what we might or might not have.